ehitchhiking-ui/src/app/shared/common/styles/fonts.sass

```diff
@@ -0,0 +1,15 @@
+$main-font: Montserrat-Regular, Cambia
+$light-font: Montserrat-Light, Cambia
+$bold-font: Montserrat-Bold, Cambia
+
+$display-1: 5rem
+$display-2: 4rem
+$display-3: 3rem
+$headline: 2.8rem
+$title: 2.6rem
+$subheading-1: 2.4rem
```

Comment:
pls add body-1 and body-2 because they are declared in https://material.angular.io/guide/typography

Reply:
done